README.md

```diff
@@ -3,12 +3,12 @@
 [![codecov](https://codecov.io/gh/skaut/Skautske-hospodareni/branch/master/graph/badge.svg?token=4qqp9q95cF)](https://codecov.io/gh/skaut/Skautske-hospodareni)
 [![FOSSA Status](https://app.fossa.com/api/projects/git%2Bgithub.com%2Fskaut%2FSkautske-hospodareni.svg?type=shield)](https://app.fossa.com/projects/git%2Bgithub.com%2Fskaut%2FSkautske-hospodareni?ref=badge_shield)
 
-**Doporučujeme přečíst si, [jak funguje vývoj a plánování práce](.docs/planovani-prace.md).**
+**Doporučujeme přečíst si, [jak funguje vývoj a plánování práce](docs/readme/planovani-prace.md).**
 Předejdeme tím případným nedorozuměním.
 
 Pro zapojení do vývoje doporučujeme přečíst:
```

Comment:
proc ta mezi složka `/readme/` ?

Reply:
Asi půjde odstranit. Mrknu na to.

Reply:
Pokud tam nebude vitepress, je to irelevantní

Reply:
stejne to bude v nejake slozce ne?

Reply:
Ano, udělám revert a nechám to tak, jak to bylo